src/test/ui/async-await/in-trait/fn-not-async-err2.rs

```diff
@@ -0,0 +1,21 @@
+// edition: 2021
```

Comment:
Can you add a variant of this test where `return_position_impl_trait_in_trait` is added? It probably will pass.

Reply:
will try it out

Reply:
Added :)